docs/source/blog/6-costly-data-labeling-mistakes-and-how-to-avoid-them.md

```diff
@@ -0,0 +1,127 @@
+
+But what about all of the data that was already tagged? Those objects will lack the new tags, leading to inconsistent training data. 
+
+### The Fix: Lean On Domain Experts Early and Often
+
+The best possible fix is to engage subject matter experts early to help develop the taxonomy and tags. If you, the project manager, aren’t a domain expert, call one in before you build your initial tag list. The domain expert can help guide every part of the project, but their expertise is especially useful during this stage. Have you accounted for every possible label? Are there additional labels you can add? They’ll be able to tell you. 
+
+Crowdsourcing can also be really helpful during this stage. If you can think back to the last time you and a group brainstormed a list of ideas, you’ll notice that group lists usually capture more information than any one person can offer. It’s the same idea here: More heads are better than one. Seek input from the other people working on the project, from the engineers to the annotators themselves. 
+
+Even with careful planning, it’s often the case that annotators will discover the need for new labels along the way. With Label Studio, administrators can give annotators the ability to add new labels while working on a task to improve accuracy.
+
+## 4. Overwhelming Tag Lists
+
+Of course, it’s also possible for project managers to arm their annotators with too many initial tags. Sifting through hundreds of tags can overwhelm your annotators, leading to longer, costlier projects. 
+
+Guided by fun heuristics like the [availability bias](https://catalogofbias.org/biases/availability-bias/), the annotators will then resort to using tags from the beginning of the list instead of scouring the entire list and finding the perfect label every time. This results in training data that doesn’t tell a complete or accurate story about the raw dataset. 
+
+Colossal tag lists also force annotators to become increasingly subjective in their choices—which can introduce more room for error. If you’re asked to label a blue square and your tag options are *red, yellow, blue,* you’ll pick the label “blue.” On the other hand, if you’re served a blue square and your tag options are *cerulean, cobalt, navy, aqua, aquamarine, royal blue, midnight blue,* you’ll have to start making decisions that might feel, to you, somewhat arbitrary. 
+
+The more shades of gray (or blue) you add to a project, the more likely you are to end up with redundant and inconsistent training data.
+
+### The Fix: Use Broad Classes With Subtasks
+
+In most cases, it’s a good idea to organize your tags around the broadest possible topics. However, there are certain projects that do require very granular labels. ImageNet famously uses [over 100,000 labels](https://image-net.org/about.php)—it doesn't just want its AI to recognize dogs; it wants it to recognize Great Danes, Dalmatians, and Water Spaniels. 
```

Comment:
`However, there are certain projects that do require very granular labels.`  => seems wordy. 

Replace it with the following:
However, certain projects that do require very granular labels.